docs/interfaces/index.md

```diff
@@ -69,17 +67,25 @@ make install -j4
 ```
 pip3 install <acados_root>/interfaces/acados_template
 ```
+Note: Use `pip` instead of `pip3` if you are installing in a virtual Python environment.
```

Comment:
using a virtual env is not a necessary not sufficient condition for switching between pip and pip3 as far as I know

Reply:
Using a virtual env should be sufficient to use `pip`.
If you run `which pip`, it points to the one in the virtual env.
Or am I missing something? :thinking:

Reply:
Depending on the installation of pip you could have different aliasing. Indeed on Ubuntu when you use a virtualenv pip -> pip3, but I would not be surprised if this was not the case with other environment manager/OS combinations (please prove me wrong :))

Reply:
> with other environment manager/OS combinations

Okay, I wont look into these combinations :sweat_smile: 
I made the note more generic:

Note: If you are working with a virtual Python environment, use the `pip` corresponding to this Python environment instead of `pip3`.